scripts/migrate/export-legacy-papers.py

```diff
@@ -0,0 +1,159 @@
+
+            pdf = doc.get('arXivID')
+            if (pdf is not None):
+                pdf = 'https://arxiv.org/abs/' + pdf
+            else:
+                # strip off the little something extra at the front of the URL field
+                pdf = doc.get('url', '???').replace('URL~', '')
+            if DEBUG: print "PDF: " + pdf
+
+            tauthor = None
+            authorNames = []
+            authorEmails = []
+            if DEBUG: print("\t\tAuthors: " + authors['name'])
+            for sub in authors['subscribers']:
+                user = legacyUsers.find_one({"_id" : sub})
+                if DEBUG: print '\t\t\t' + user['fullname']
+                authorNames.append('"' +  user['fullname'] + '"')
+                # take the first email for this author (they can specify multiple emails)
+                # We may want to check that it's verified or the preferred email
+                firstEmail = next(iter(user['emails'] or []), None)
+                authorEmails.append('"' + str(firstEmail['email']) + '"')
+                # assume the first person listed is the "true author"
+                if tauthor is None:
+                    tauthor = firstEmail['email']
+
+            authorEmailList = ', '.join(authorEmails)
+            authorNameList = ', '.join(authorNames)
+
+            dttm = doc['created'];
+            t =  datetime.datetime(dttm.year, dttm.month, dttm.day, dttm.hour, dttm.minute)
+            createddt = str(time.mktime(t.timetuple()) * 1000)
+
+            forumid = shortid.ShortId().generate()
+
+            # note: for now, we'll just use count for the paper number, normally this
+            # would come from the "counter" collection in Arango
+
+            # rather than replacing double quotes with '\"', we'll replace them with a
+            # single quote. There is some data that the escaped double quote wouldn't work for
+            # such as 'na\"ive'. So rather than handle all special cases, just use a single quote.
+            abstract = doc['summary'].replace('"', '\'')
+            # the import process to Arango wants one JSON record per line.
+            noteJSON = ('{'
+                '"tddate": null,'
+                '"replyto": null,'
+                '"active": true,'
+                '"tmdate": ' + createddt + ','
+                '"tcdate": ' + createddt + ','
+                '"number": ' + str(count) + ','
+                '"id": "' + forumid + '",'
+                '"invitation": "' + submitInvite + '",'
+                '"forum":   "' + forumid + '",'
+                '"tauthor":  "' + tauthor + '",'
+                '"signatures": [],'
+                '"readers": ["everyone"],'
+                '"writers": [],'
+                '"content": {'
+                '"decision" : "' + endorsement + '",'
+                '"title": "' + doc['title'] + '",'
+                    '"abstract": "' + abstract + '",'
+                    '"pdf":  "' + pdf + '",'
```

Comment:
If I change the pdf field to have the value "/pdf/<existing id>.pdf" the link will render correctly: /pfg/<forum id>.pdf.

We could then just copy the PDFs from /file to the /pdf path and everything should work without having to invoke the API which would break existing links to PDFs and could result in duplicate files if we run this multiple times.